frontend/readhistory.lua

```diff
@@ -40,14 +41,17 @@ local function timeFirstOrdering(l, r)
 end
 
 function ReadHistory:_indexing(start)
+    assert(self ~= nil)
```

Comment:
I don't think that makes a difference to the user? Otherwise I don't really see how that's relevant to the question. :-)